static/src/server.ts

```diff
@@ -297,25 +301,22 @@ function getBlockTileResults(
               useChartUrl
             )
           );
-          break;
-        /* TODO: foreignobject doesn't work with the svg to png converter.
-                  Need to find a different way to render an svg of the ranking
-                  table & then re-enable this.
+          break;*/
```

Comment:
maybe in future we might want a "client=" and decide based on that.  have a TODO?

Reply:
added the client param